console-ui/src/layouts/MainLayout.js

```diff
@@ -146,7 +150,12 @@ class MainLayout extends React.Component {
                 )}
               </div>
             </div>
-            <div className="right-panel next-shell-sub-main">{this.props.children}</div>
+            <div className="right-panel next-shell-sub-main">
+              {authEnabled === 'false' ? (
+                <Message type="notice">{this.props.notice}</Message>
```

Comment:
<Message type="notice">
<div
      dangerouslySetInnerHTML={{__html: this.props.notice}}
    />
</Message>